internal/kafka/command_cluster_create.go

```diff
@@ -171,6 +172,17 @@ func (c *clusterCommand) create(cmd *cobra.Command, args []string) error {
 		setClusterConfigCku(&createCluster, int32(cku))
 	}
 
+	if cmd.Flags().Changed("network") {
+		networkId, err := cmd.Flags().GetString("network")
+		if err != nil {
+			return err
+		}
+		if clusterType != skuDedicated {
+			return errors.NewErrorWithSuggestions("the `--network` flag can only be used when creating a dedicated Kafka cluster", "Specify a dedicated cluster with `--type`.")
+		}
+		setClusterNetwork(&createCluster, cmkv2.NewEnvScopedObjectReference(networkId, "", ""))
```

Comment:
Can you manually initialize the "scoped object reference"? It's a bit confusing how we have two `""` parameters here.